test_runner/fixtures/neon_fixtures.py

```diff
@@ -1950,7 +1950,11 @@ def dump_index_part(self, path: Path) -> IndexPartDump:
         return IndexPartDump.from_json(parsed)
 
 
-class Logging:
+class AssertLogging:
+    """
+    A class that server to quickly assert statement from a logfile defined in class inheriting.
```

Comment:
nit: this comment doesn't make a lot of sense to me. How about: `A mixin class which provides utilities for inspecting the logs of a service`